CustomSecuritySample2016/README.md

```diff
@@ -156,59 +156,13 @@ Add the following <authorization> element directly after the <authentication> el
 This will deny unauthenticated users the right to access the report server. The previously established loginUrl attribute of the <authentication> element will redirect unauthenticated requests to the Logon.aspx page.
 
 
-## Step 4: Some of the other changes required in the web.config file and Microsoft.ReportingServices.Portal.WebHost.exe.config
+## Step 4: Generate Machine Keys
 
-Adding Machine Keys
+Using Forms authentication requires that all report server processes can access the authentication cookie. This involves configuring a machine key and decryption algorithm - a familiar step for those who had previously setup SSRS to work in scale-out environments.
 
--	For the case of Forms authentication which requires the decryption of the Authentication cookie, both processes need to be configured with the same machine key and decryption algorithm. This was a step familiar to those who had previously setup SSRS to work on scale-out environments, but now is a requirement even for deployments on a single machine.
+Generate and add machine keys to your RSReportServer.config file. You should use a validation key specific for you deployment, there are several tools to generate the keys like Internet Information Services Manager(IIS) another example is: http://www.a2zmenu.com/utility/machine-key-generator.aspx 
```

Comment:
This folder is for the 2006 sample, you should keep the sample for RS 2016 and remove the PBI server piece

And update the document under customSecuritySample with this changes